adapters/appnexus/appnexus.go

```diff
@@ -495,7 +495,7 @@ func preprocess(imp *openrtb2.Imp, defaultDisplayManagerVer string) (string, err
 	if appnexusExt.InvCode != "" {
 		imp.TagID = appnexusExt.InvCode
 	}
-	if appnexusExt.Reserve > 0 {
+	if imp.BidFloor <= 0 && appnexusExt.Reserve > 0 {
```

Comment:
We don't validate the BidFloor in the original request. Would it be better to just check for `0` to be sure it omitted rather than explicitly set to a nonsensical value?

Reply:
I think since a negative value would anyway be invalid, this check makes sense to me